src/colors.ts

```diff
@@ -15,3 +15,21 @@ const colors: string[] = [colorPrimary, colorYellow, colorPurple, colorTeal, col
 export function getColor(index: number) {
     return colors[index % colors.length];
 }
+
+export function getColorForResourceType(type: 'JSON' | 'GeoJSON' | 'CSV' | 'Embedded') {
+    switch (type) {
+        case 'JSON': {
+            return 'var(--type-color-json)';
```

Comment:
Please add `break` to the cases (I know it won't make much of a difference in this case)